### ISSUE TITLE
p-table resize column does not work with pagination.

### ISSUE THREAD

**Issue by pachu4u** [primefaces/primeng#8837]
If you have a PrimeNG PRO Support subscription please post your issue at;

https://pro.primefaces.org 

where our team will respond within 4 business hours.

If you do not have a PrimeNG PRO Support subscription, fill-in the report below. Please note that
your issue will be added to the waiting list of community issues and will be reviewed on a first-come first-serve basis, as a result, the support team is unable to guarantee a specific schedule on when it will be reviewed. Thank you for your understanding.

Current Queue Time for Review
Without PRO Support: ~8-12 weeks.
With PRO Support: 1 hour

**I'm submitting a ...**  (check one with "x")
```
[X ] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
Please demonstrate your case at stackblitz by using the issue template below. Issues without a test case have much less possibility to be reviewd in detail and assisted.

https://stackblitz.com/github/primefaces/primeng-issue-template

**Current behavior**

Resize columns does not seem to work when the pagination is turned on. I had a working resize columns with virtual scrolling. But as soon as I switched from virtual scroll to pagination it stopped working. No example either in the official documentation.

`<p-table [columns]="visibleColumns" [value]="result" [rows]="api.companies.size" paginatorPosition="top"
                    (onLazyLoad)="loadDataOnScroll($event)" [resizableColumns]="false" [lazy]="true" 
                    [showCurrentPageReport]="true" resetPageOnSort="true" (onPage)="paginate($event)" [first]="first"
                    currentPageReportTemplate="Showing {first} to {last} of {totalRecords} entries" [totalRecords]="totalRecords"
                    [paginator]="true" class="customTableStyle" [loading]="loading">`

`<ng-template pTemplate="colgroup" let-columns>
                        <colgroup>
                            <col *ngFor="let col of columns" [style.width]="col.width">
                        </colgroup>
                    </ng-template>`

Header template
`<th *ngFor="let col of columns" [pSortableColumn]="col.prop" pResizableColumn >`

Body template
`<td *ngFor="let col of columns" class="ui-resizable-column">`

**Expected behavior**

Column should be resizable
**Minimal reproduction of the problem with instructions**
Setup a table with pagination and resizable columns

**What is the motivation / use case for changing the behavior?**

**Please tell us about your environment:**

* **Angular version:** 5.X

8.2.3
* **PrimeNG version:** 5.X

9.0.3
* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]

 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

* **Node (for AoT issues):** `node --version` =

**Comment by sarisen** [comment]
I think there is no problem.  resizableColumns must be true

[https://stackblitz.com/edit/github-hqfvuv](url)

**Comment by pachu4u** [comment]
@sarisen - Sorry resizableColumns was true. I just turned it off as it wasn't working. I should have corrected it before posting.

Anyway I have modified the stackblits with scroll and custom width, which works fine.

https://stackblitz.com/edit/github-hqfvuv-dicyes

Now if you convert this example to pagination it does not work. Looks like as soon as I add below code to your example it stops working.

`<ng-template pTemplate="colgroup" let-columns>
            <colgroup>
                <col *ngFor="let col of columns" [style.width]="col.width">
            </colgroup>
        </ng-template>`

https://stackblitz.com/edit/github-hqfvuv-k4dfcm - Modified example with pagination with custom width, which does not work.
